services/commands-service.ts

```diff
@@ -33,6 +38,7 @@ export class CommandsService implements ICommandsService {
 	}
 
 	public async executeCommandUnchecked(commandName: string, commandArguments: string[]): Promise<boolean> {
+		this.commands.push({ commandName: commandName, commandArguments: commandArguments });
```

Comment:
You can use the short syntax here `this.commands.push({ commandName, commandArguments });`